ci-operator/step-registry/cucushift/installer/check/mirror-registry-services/cucushift-installer-check-mirror-registry-services-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "cucushift/installer/check/mirror-registry-services/cucushift-installer-check-mirror-registry-services-ref.yaml",
```

Comment:
How about moving the step to `ci-operator/step-registry/mirror-images/` ?

Reply:
Moved to `mirror-images-check-registry-service`